tests/foreman/cli/test_remoteexecution.py

```diff
@@ -494,6 +494,7 @@ def test_positive_run_scheduled_job_template_by_ip(self, rex_contenthost, target
 
     @pytest.mark.tier3
     @pytest.mark.upgrade
+    @pytest.mark.skip("Receptor plugin is deprecated/removed for Satellite >= 6.11")
```

Comment:
Are you planning to fix this for RHC in the future? If yes, good, I think you should do that (especially since it probably requires little changes, similar to the UI test). If not, is there any reason not to delete the test entirely?

Reply:
yes, I'm planning to fix it in future.